shared/src/sources/uniswap/pool_fetching.rs

```diff
@@ -13,6 +13,10 @@ use std::{collections::HashSet, sync::Arc};
 pub const MAX_BATCH_SIZE: usize = 100;
 const POOL_SWAP_GAS_COST: usize = 60_000;
 
+lazy_static::lazy_static! {
+    static ref POOL_MAX_RESERVES: U256 = U256::from((1u128 << 96) - 1);
```

Comment:
I didn't even realize pools had max reserves? Is this specific to uniswap since they use uint96 for amounts?

Reply:
In Balancer they have `uint256` reserves, so a higher maximum.

Reply:
So then I would assume this change only applies to Uniswap-Like-Pools

Seems reasonable.